ApplicationLibCode/UserInterface/RiuResultTextBuilder.cpp

```diff
@@ -279,6 +278,75 @@ QString RiuResultTextBuilder::geometrySelectionText( QString itemSeparator )
     return text;
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+QString RiuResultTextBuilder::coordinatesText( const RigGridBase* grid, size_t globalCellIndex, const QString& itemSeparator )
+{
+    QString text;
+
+    bool showCenter = false;
+    bool showCorner = false;
+
+    if ( m_eclipseView )
+    {
+        std::vector<RimMultipleEclipseResults*> additionalResultSettings;
+        m_eclipseView->descendantsOfType( additionalResultSettings );
+        if ( !additionalResultSettings.empty() )
+        {
+            showCenter = additionalResultSettings[0]->showCenterCoordinates();
+            showCorner = additionalResultSettings[0]->showCornerCoordinates();
+        }
+    }
+
+    if ( showCorner || showCenter )
```

Comment:
Uneccessary to check "showCorner || showCenter" here since it is checked again later?